tools/reckless

```diff
@@ -340,6 +341,8 @@ def populate_local_dir(path: str) -> list:
 
 
 def populate_local_repo(path: str) -> list:
+    if not Path(os.path.realpath(path)).exists():
+        logging.warning(f"cannot populate {path}: does not exist")
     assert Path(os.path.realpath(path)).exists()
```

Comment:
assert will log?  Also, you can have a message as the second argument to assert...

Reply:
I like the second message idea, but after fixing the root causes of these original issues, the logging should now be superfluous so I've removed it.